docs/rules/0134/request-path-required.md

```diff
@@ -0,0 +1,59 @@
+---
+rule:
+  aep: 134
+  name: [core, '0134', request-path-required]
+  summary: Delete RPCs must have a `path` field in the request.
```

Comment:
```suggestion
  summary: Update RPCs must have a `path` field in the request.
```

I think a few more places need "Delete" replaced with "Update".

Reply:
Potentially a way to do that? I can make a bug for it.

Reply:
I'd probably refactor the linter rule to look at specific method names (Read{singular}/Update{Singular}, etc), then require the field for those methods.

Maybe it'll be kind of hard to incorporate that into multiple different rule sets though.